### DISCUSSION TITLE
Continue numbered list

### DISCUSSION THREAD

**Discussion by reefland** [TriliumNext/Trilium#1995] [Ideas]
I'd like to be able create multiple numbered listed in a note and be able to continue numbering from previous list in the note, not always start from 1.

I'd suggest when "Editing block" properties, and selecting "Numbered List" be able to use the Drop Down symbol to select option to "Continue Numbering" or "Start from 1".  

PS: If ya like allow a custom start number, I don't have this requirement, but I'm sure it could be helpful to others.

**Comment by zadam** [comment]
Hi, Trilium uses [CKEditor 5](github.com/ckeditor/ckeditor5/) as its editing component and such feature would have to be implemented there.
